### PR TITLE
Fix node fewshots data generation logic error.

### PR DESCRIPTION
**Pull request by AzulEterno**
Fix node fewshots data generation logic error.

Fixed non dict data as final result for best_params error.

### CODE REVIEW

**Comment by 6niezhiping6** [comment]
Why is it that after I use the pre-trained model you provided, the loss when fine-tuning stays almost the same, it's all around 1.9

**Comment by 6niezhiping6** [comment]
Why is it that after I use the pre-trained model you provided, the loss when fine-tuning stays almost the same, it's all around 1.9
